starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +23,49 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(words) {
+  var longestWord = '';
+  if (words.length > 0) {
```

Comment:
This is a way of returning undefined when the array is empty. Another one is:
```
var longestWord = words[0]
```
This way, you asign to your comparator (longestWord) the first array's element. If the array is empty, longestWord will be 'undefined' and the function will return 'undefined'. Using an internal element for comparision solve some tricky problems (e.g. finding a minimum).
```
function findLongestWord(words) {
  var longestWord = words[0];

  for (var i = 1; i < words.length; i++) {
    if (words[i].length > longestWord.length) {
      longestWord = words[i];
    }
  }
  return longestWord;
}
```